### PR TITLE
Compatibility with connector v0.1

### PR DESCRIPTION
**Pull request by splix**

### CODE REVIEW

File: electron/services.js [hunk 1]
```diff
@@ -37,11 +37,25 @@ export class Services {
     }
 
     start() {
-        return this.startRpc()
-            .then(this.startConnector.bind(this))
-            .catch((err) => {
-                log.error('Failed to run services', err);
-            });
+        return Promise.all([
```
Comment (whilei) [inline electron/services.js]: Nice.

File: package.json [hunk 2]
```diff
@@ -105,6 +105,7 @@
     "css-loader": "0.28.0",
     "electron": "^1.6.2",
     "electron-packager": "^8.6.0",
+    "devtron": "1.4.0",
```
Comment (whilei) [inline package.json]: What is devtron for?

File: src/store/accountActions.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { rpc } from 'lib/rpc';
 import { getRates } from 'lib/marketApi';
 import { address } from 'lib/validators';
 import { loadTokenBalanceOf } from './tokenActions';
+import log from 'loglevel';
```
Comment (elaineo) [inline src/store/accountActions.js]: let's comment out lines 17-22 in this file (load tokenBalance)
sorry, there was not a better place to put this comment :)

Comment (splix) [inline src/store/accountActions.js]: I see, thank you

File: package.json (hunk 2 above)
Comment (splix) [inline package.json]: additional tab to Chrome DevTools, shows electron IPC calls and other stuff

**Pull request merged**